Analysis/include/AnaTupleReader.h

```diff
@@ -22,10 +22,11 @@ class AnaTupleReader {
 
     AnaTupleReader(const std::string& file_name, Channel channel, NameSet& active_var_names,
                    const std::vector<std::string>& input_friends, const EventTagCreator& event_tagger, const std::string& _mdnn_version,
-                    std::set<UncertaintySource>& _norm_unc_sources);
+                    std::set<UncertaintySource>& _norm_unc_sources, int& hastune);
```

Comment:
- why `hastune` is a reference?
- presence `hastune` is a workaround, I would keep it local in AnaTupleReader and don't propagate any modifications to the config etc.